website/docs/r/vcenter.html.markdown

```diff
@@ -56,7 +56,7 @@ The following arguments are supported:
 
 The following attributes are exported on this resource:
 
-* `has_proxy` - Indicates that a proxy exists within vCloud Director that proxies this vCenter
+* `has_proxy` - Indicates that a proxy exists within Tenant Manager that proxies this vCenter
```

Comment:
I just noticed this from previous PR. "Buttery butter" here :) Maybe we could rephrase it to actually provide some ellaboration.

Reply:
The origin is this, however here it is a read-only flag remaining https://techdocs.broadcom.com/us/en/vmware-cis/cloud-director/vmware-cloud-director/10-6/vmware-cloud-director-service-provider-portal-guide-10-6/managing-infrastructure-resources-admin/accessing-vsphere-components-through-vcd-endpoints-and-proxies-admin.html

TBH I don't think this is yet support and can perhaps be undocumented

Reply:
Improved the wording to be more comprehensive